local operations = {
	add_attribute = require(script.add_attribute),
	add_tag = require(script.add_tag),
	copy_instance = require(script.copy_instance),
	create_instance = require(script.create_instance),
	delete_instance = require(script.delete_instance),
	deselect_instance = require(script.deselect_instance),
	duplicate_instance = require(script.duplicate_instance),
	execute_luau = require(script.execute_luau),
	get_properties = require(script.get_properties),
	get_sound_playback_info = require(script.get_sound_playback_info),
	move_node = require(script.move_node),
	paste_instance = require(script.paste_instance),
	play_sound = require(script.play_sound),
	redo = require(script.redo),
	remove_attribute = require(script.remove_attribute),
	remove_tag = require(script.remove_tag),
	rename_attribute = require(script.rename_attribute),
	rename_instance = require(script.rename_instance),
	set_attribute = require(script.set_attribute),
	set_property = require(script.set_property),
	set_sound_time_position = require(script.set_sound_time_position),
	stop_sound = require(script.stop_sound),
	undo = require(script.undo),
}
return operations